tue-documentation/setup

```diff
@@ -10,15 +10,19 @@ function tue-make-documentation
 {
     if [ -n "$TUE_ROS_DISTRO" ] && [ -d "$TUE_SYSTEM_DIR" ]
     then
-        local build_tool
-        build_tool=$(cat "$TUE_SYSTEM_DIR"/devel/.built_by)
+        local build_tool=""
+        if [ -f "$TUE_SYSTEM_DIR"/devel/.built_by ]
```

Comment:
Why is this addition necessary?

Reply:
So you don't get the error if the file doesn't exists. This is the case on a new pc or after cleaning your ws.